### DISCUSSION TITLE
How to get the cashout information from a bet

### DISCUSSION THREAD

**Discussion by kam06** [cloudbet/docs#51] [General]
Hello all, 

I would like to know please what API to use to get the cashout of my bet ? 

Thanks

**Comment by kgravenreuth** [comment]
Hi @kam06 

Thanks for your request to add cashout bets to the public betting API. We're considering it for our roadmap, while cashout betting is not available yet on our Public Trading API. You are welcome to place cashout bets using our Web interface at www.cloudbet.com

Furthermore, as mentioned in #31 please note that for bets placed using our Trading API, they will only be available via our [Bet history endpoint](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=social&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistoryV4) and are not visible on our Web interface. 

Cheers,
Klaus

**Comment by kam06** [comment]
Hi @kgravenreuth 

Thanks for your answer. 
When do you think the cashout betting will be available on your Public Trading API please ? 

Thanks
Regards
